src/main.rs

```diff
@@ -15,6 +36,31 @@ fn main() {
         }
     };
 
+    let debug = match std::env::var("REVAULTGUI_DEBUG") {
+        Ok(var) => match FromStr::from_str(&var) {
+            Ok(v) => v,
```

Comment:
Just to be sure, so Rust here knows that you want a bool based on the last Err() branch below ?

Reply:
Yes, with the  branch `Err(VarError::NotPresent) => false,`